Assets/Scripts/UIScripts/PauseScreen.cs

```diff
@@ -43,8 +43,8 @@ public void Resume()
     {
         pauseMenuUI.SetActive(false);
         settingsMenuUI.SetActive(false);
-        MidiManager.current.ResumePlayback();
-        if (Time.timeSinceLevelLoad > 5)
+        //MidiManager.current.ResumePlayback();
+        if (Time.time > 5)
```

Comment:
Why is this change necessary? I believe @judy-n changed this to timeSinceLevelLoad to avoid issues when playtesting with a build (instead of the unity editor) (since Time.time does not reset to 0 when a scene loads/reloads)

Reply:
yeah exactly, everywhere that uses `Time.time > 5` should be `Time.timeSinceLevelLoad > 5` . It fixes a bug we had, like if you waited 5 seconds in the main menu screen then this wait will never happen unless you use "timeSinceLevelLoad"